include/beman/inplace_vector/inplace_vector.hpp

```diff
@@ -546,7 +571,10 @@ class inplace_vector : public inplace_vector_base<T, Capacity> {
 
   template <class... Args>
   constexpr reference unchecked_emplace_back(Args &&...args) {
-    auto final = std::construct_at(end(), std::forward<Args>(args)...);
+    // TODO: what is the feature flag for std::construct_at ??
+    // auto final = std::construct_at(end(), std::forward<Args>(args)...);
+    // This is just construct_at expanded out.
+    auto final = ::new (end()) T(std::forward<Args>(args)...);
```

Comment:
How do I selectively use `construct_at` instead of placement new?

`construct_at` doesn't have a feature-test macro.

Reply:
Looks like it could be `__cpp_lib_constexpr_dynamic_alloc` or `__cpp_constexpr_dynamic_alloc` as `construct_at` was added in [P0784R7](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p0784r7.html)

Reply:
No problem!

And no... at least not that I'm aware of off the top of my head. I would have to do some digging to find out.

Reply:
Alright I guess constexpr stays at C++17 then.